api/app/signals/apps/reporting/csv/utils.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+from django.conf import settings
+from django.core.files.storage import FileSystemStorage
+from swift.storage import SwiftStorage
+
+
+def _get_storage_backend(swift_parameters):
```

Comment:
This could be a util on a higher level. We can use this in other situations as well

Reply:
agree will make a ticket in the backlog (SIG-1811)